starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -686,6 +685,11 @@ sub check_dither {
     my $obs_tstart = $self->{obs_tstart};
     my $obs_tstop = $self->{obs_tstop};
 
+    unless (defined $obs_tstart){
```

Comment:
This is an unrelated bugfix, but it was coming up the replan/reopn weeks I was using to look at probabilities, and it is harder to check that the probability fixes are right when there is a bunch of useless error reporting at console.